include/seqan3/alignment/matrix/detail/affine_cell_proxy.hpp

```diff
@@ -145,7 +157,11 @@ class affine_cell_proxy : public tuple_t
     //!\brief Converting constructor. Initialises from another tuple type.
     template <typename other_tuple_t>
     //!\cond
-        requires std::constructible_from<tuple_t, other_tuple_t &&>
+#ifdef SEQAN3_WORKAROUND_GCC_96922
+        requires (constructible_from_aux<tuple_t, other_tuple_t &&>)
+#else // workaround ^^^|vvv no workaround
+        requires (std::constructible_from<tuple_t, other_tuple_t &&>)
```

Comment:
because it deviates from the suggested style to always wrap the requires expression in parenthesis. And then the question remains, why that was not done here but in all other places. Although being quite verbose it does provide us the right information regarding this issue.